### DISCUSSION TITLE
Wondering about plans for Home Assistant work

### DISCUSSION THREAD

**Discussion by wonderslug** [G-Two/subarulink#76] [General]
Howdy @G-Two 

I was wondering what you had plans for with regards to more Home Assistant integration?

If you don't already have things in the works for remote start/stop, lights, and horn I was thinking of putting up a PR for them.

Thanks for getting the base level g3 support in here recently as well.

**Comment by stboch** [comment]
@wonderslug there is a HACS project that has all the features that are in the release of Home-Assistant plus more. I think you will find what you need there. https://github.com/G-Two/homeassistant-subaru

**Reply by wonderslug** [reply]
Perfect thanks @stboch   exactly what I was looking for.
